pipelines/prep_data.py

```diff
@@ -37,6 +42,8 @@ def py_scalar_to_r_scalar(py_scalar):
 
     state_abb = state_abb if state_abb != "US" else "USA"
 
+    temp_file = Path(temp_dir, "nhsn_temp.parquet")
```

Comment:
Do we need to check if this exists already and error?

Reply:
because we explicitly created the enclosing `temp_dir` above via https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp this should be safe without that check